EIPS/eip-1.md

```diff
@@ -4,7 +4,6 @@ title: EIP Purpose and Guidelines
 status: Active
 type: Meta
 author: Martin Becze <mb@ethereum.org>, Hudson Jameson <hudson@ethereum.org>, and others
-        https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1.md
```

Comment:
It's not clear what purpose this URL served here.

Reply:
I have changed this file locally just never submitted a PR. My understanding it tries to signal "look at the history".

What I opted for was "Martin ..., Hudson ..., et al."

Reply:
Oh I see, it was supposed to be a part of the authors field.

Reply:
Can we change the authors for EIP-1 to just "Contributors People Like You".

Reply:
I don't have a strong feeling towards this, but IMO it's outside the scope of this PR.